WeightProducer/python/MCSampleValues.py

```diff
@@ -84,422 +84,624 @@ class MCSampleValuesHelper():
+                                      XS_2018=156.5, XSSource_2018="Same as 2017"),
         },
         "QCD_Pt_800to1000" : {
-            "CrossSection" : XSValues(XS_2016=32.293),
+            "CrossSection" : XSValues(XS_2016=32.293, XSSource_2016="AN2017_013_v17",
+                                      XS_2017=26.28, XSSource_2017="XSDB",
+                                      XS_2018=26.28, XSSource_2018="Same as 2017"),
         },
         "QCD_Pt_1000to1400" : {
-            "CrossSection" : XSValues(XS_2016=9.4183),
+            "CrossSection" : XSValues(XS_2016=9.4183, XSSource_2016="AN2017_013_v17",
+                                      XS_2017=7.47, XSSource_2017="XSDB",
+                                      XS_2018=7.47, XSSource_2018="Same as 2017"),
         },
         "QCD_Pt_1400to1800" : {
-            "CrossSection" : XSValues(XS_2016=0.84265),
+            "CrossSection" : XSValues(XS_2016=0.84265, XSSource_2016="AN2017_013_v17",
+                                      XS_2017=0.6484, XSSource_2017="XSDB",
+                                      XS_2018=0.6484, XSSource_2018="Same as 2017"),
         },
         "QCD_Pt_1800to2400" : {
-            "CrossSection" : XSValues(XS_2016=0.114943),
+            "CrossSection" : XSValues(XS_2016=0.114943, XSSource_2016="AN2017_013_v17",
+                                      XS_2017=0.08743, XSSource_2017="XSDB",
+                                      XS_2018=0.08743, XSSource_2018="Same as 2017"),
         },
         "QCD_Pt_2400to3200" : {
-            "CrossSection" : XSValues(XS_2016=0.00682981),
+            "CrossSection" : XSValues(XS_2016=0.00682981, XSSource_2016="AN2017_013_v17",
+                                      XS_2017=0.005236, XSSource_2017="XSDB",
+                                      XS_2018=0.005236, XSSource_2018="Same as 2017"),
         },
         "QCD_Pt_3200toInf" : {
-            "CrossSection" : XSValues(XS_2016=0.000165445),
+            "CrossSection" : XSValues(XS_2016=0.000165445, XSSource_2016="AN2017_013_v17",
+                                      XS_2017=0.0001357, XSSource_2017="XSDB",
+                                      XS_2018=0.0001357, XSSource_2018="Same as 2017"),
         },
-        # DY/Z: k-factor of 1.23 applied
+        # DY/Z
         "DYJetsToLL_M-50_HT-100to200" : {
-            "CrossSection" : XSValues(XS_2016=181.302),
-            "kFactor" : kFactorValues(kFactor_2016=1.23),
+            "CrossSection" : XSValues(XS_2016=147.4, XSSource_2016="XSDB",
+                                      XS_2017=161.1, XSSource_2017="XSDB",
+                                      XS_2018=161.1, XSSource_2018="Same as 2017"),
+            "kFactor" : kFactorValues(kFactor_2016=1.23, kFactorSource_2016="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#DY_Z",
+                                      kFactor_2017=1.165, kFactorSource_2017="NNLO/LO=6225.42/5343",
```

Comment:
is the zinv team on board with this kfactor?

Reply:
also, if this is calculated from inclusive xsecs, I repeat my statement of the physical impossibility of changing fundamental cross sections

Reply:
(ignore the above comment)